utils/fieldToConceptIdMapping.js

```diff
@@ -272,42 +272,57 @@ module.exports = {
     started:        615768760,
     submitted:      231311385,
     dataDestruction: {
-        consentFirstname: 471168198,
-        consentMiddleName: 436680969,
-        consentLastName: 736251808,
-        consentSuffixName: 480305327,
-        birthMonth: 564964481,
-        birthDay: 795827569,
-        birthYear: 544150384,
+        ConsentFirstname: 471168198,
+        ConsentMiddleName: 436680969,
```

Comment:
I notice case changes on a lot of these constants, and a number of constants which have been deleted. Can you confirm that you've double-checked that none of these constants were used elsewhere in connectFaas?

Reply:
Yup, i followed this docs https://nih.app.box.com/file/839951579186 and updated it.

Reply:
I don't see why that document can change conventions in JavaScript.